src/main/java/fr/xephi/authme/process/logout/AsynchronousLogout.java

```diff
@@ -53,6 +53,7 @@ public void logout(Player player) {
 
         playerCache.removePlayer(name);
         database.setUnlogged(name);
+        database.revokeSession(name);
```

Comment:
It looks like `revokeSession` and `setUnlogged` are always called together. If we merge this into one call we would save some DB operations, which would be good.

Reply:
Nope, we don't do that when the player quit the server.